Исправление орфографических ошибок 1-3 ЛР/Р. Мокеева Ирина РТ-31.

irinamokeeva510-creator opened the pull request:
Добрый день, внесла правки орфографических ошибок. 
Мокеева Ирина РТ-31.

HepoH3 commented on Labs/4. Primitive programmable device/Индивидуальное задание/README.md:
```diff
@@ -176,91 +176,126 @@ cyberconverter принимает до двух аргументов. Поряд
-Результат вычислений: `out_o = 0...0101110`.
+Результат вычислений: `out_o = 0...0101110`.  
 
+4. Поменять местами [7:0] и [15:8] биты числа `sw_i`. Вывести результат в `out_o`.  
```
```suggestion
4. Поменять местами `[7:0]` и `[15:8]` биты числа `sw_i`. Вывести результат в `out_o`.  
```

HepoH3 commented on Labs/4. Primitive programmable device/Индивидуальное задание/README.md:
```diff
@@ -176,91 +176,126 @@ cyberconverter принимает до двух аргументов. Поряд
+Результат вычислений: `out_o = 0...0101110`.  
 
+4. Поменять местами [7:0] и [15:8] биты числа `sw_i`. Вывести результат в `out_o`.  
+Пример: `sw = 0...10100000_1110010`.  
```
```suggestion
Пример: `sw_i = 0...10100000_1110010`.  
```

HepoH3 commented on Labs/4. Primitive programmable device/Индивидуальное задание/README.md:
```diff
@@ -176,91 +176,126 @@ cyberconverter принимает до двух аргументов. Поряд
+Пример: `sw = 0...10100000_1110010`.  
+Результат вычислений: `out_o = 0...11100101_10100000`.   
+
+5. Через инструкции с регистрового файла загружается два числа: `a, b`. Найти: `max + min/2`, где `max` и `min` — это соответственно максимальное и минимальное число из заданных через инструкции.   
+Пример: `a = 0...10`, `b = 0...11`.   
+Результат вычислений: `out_o = 0...100`.   
```
```suggestion
5. 5. Вычислить приблизительное значение длины вектора `[a;sw_i]`. Вычисляется как `max + min/2`, где `max` и `min` — это большее и меньшее из чисел `a` и `sw_i` соответственно.  
Пример: `a = 0...11`, `sw_i = 0...100`.   
Результат вычислений: `out_o = 0...101`.   
```

HepoH3 commented on Labs/4. Primitive programmable device/Индивидуальное задание/README.md:
```diff
@@ -176,91 +176,126 @@ cyberconverter принимает до двух аргументов. Поряд
 
-5. Если `sw_i[1:0] == 00`, то в `out_o` выводится `a`,  если `sw_i[1:0] == 01`, то `b`,  если `sw_i[1:0] == 10`, то `c`,  если `sw_i[1:0] == 11`, то `d`.
-Пример: `a = 00`, `b = 10`, `c = 11`, `d = 01`, `sw_i = 01`. Результат вычислений: `out_o = 10`.  
+8. Посчитать длину окружности при заданном радиусе `sw_i`, если `pi = 3`. Вывести результат в `out_o`.   
```
```suggestion
8. Посчитать длину окружности при заданном радиусе `sw_i`, считая что `π = 3`. Вывести результат в `out_o`.   
```

HepoH3 commented on Labs/4. Primitive programmable device/Индивидуальное задание/README.md:
```diff
@@ -176,91 +176,126 @@ cyberconverter принимает до двух аргументов. Поряд
-6. Если `sw_i` является степенью двойки, то вывести `out_o =  1`, в противном случае, `out_o = 0`.
-Пример 1: `sw_i = 100`. Результат вычислений: `out_o = 1`.
+9. Если `sw_i` является степенью двойки, то вывести `out_o =  1`, в противном случае, `out_o = 0`.   
+Пример 1: `sw_i = 100`. Результат вычислений: `out_o = 1`.   
```
В этом примере можно подумать, что вот мы щас знакорасширим как в лабе sw_i=100 и получится 1111...100, что не является степенью двойки — а out_o почему-то равен 1.  
Я к тому, что надо хотя бы ведущий бит знаковым оставлять, если не охота заморачиваться с многоточиями

HepoH3 commented on Labs/4. Primitive programmable device/Индивидуальное задание/README.md:
```diff
@@ -176,91 +176,126 @@ cyberconverter принимает до двух аргументов. Поряд
 
-23. Вычислить `a * IN` с использованием операций сложений и сдвига ("в столбик"). Вывести результат в `out_o`. [Пример](https://studfile.net/preview/8921792/page:5/)
+27. Вычислить `a * sw_i` с использованием операций сложений и сдвига ("в столбик"). Вывести результат в `out_o`.   
+[Пример](https://studfile.net/preview/8921792/page:5/)  
```
Я бы оставил ссылку на что-то более стабильное.

```suggestion
[Пример](https://en.wikipedia.org/wiki/Binary_multiplier#Binary_long_multiplication):  
```

и еще вот этот пример бы добавил оттуда же

```text
       1011   (11 в двоичном виде)
     x 1110   (14 в двоичном виде)
     ======
       0000   (это 1011 x 0)
      1011    (это 1011 x 1, сдвинутое на 1 влево)
     1011     (это 1011 x 1, сдвинутое на 2 влево)
  + 1011      (это 1011 x 1, сдвинутое на 2 влево)
  =========
   10011010   (154 в двоичном виде)
```

irinamokeeva510-creator committed:
Update README.md

исправлены две ошибки 
1)Ошибки:«увосьмиренной разрядности»
Лучше: «увосьмеренной разрядности» 
2)«Подробней»
Нормативно: «Подробнее».

irinamokeeva510-creator committed:
Update README.md

1) «…производить над входными данным…»
Правильно: «…производить над входными данными…» (Т.п., мн.ч.)
2) грамматика в формулировках условий таблиц:
«не входящим в эту таблицу» лучше: «не входящем» (о коде операции в ед. числе) (в двух строках одна и та же ошибка)

irinamokeeva510-creator committed:
Update README.md

logic [19:0] memory3 [15:0];  // объявлено
logic [19:0] memory3 [1:16];  // объявлено ещё
Имя memory3 используется дважды — это синтаксическая ошибка в реальном коде,второе объявление должно быть memory4 или другим именем.

HepoH3 committed:
Унификация слова "подробнее".

HepoH3 committed:
Ёфикация

HepoH3 committed:
ЛР2. Уточнение формулировки
